dataactcore/config_example.yml

```diff
@@ -98,86 +138,82 @@ services:
     # same path as the broker_files setting (above)
     # for simplicity.
     # This should be an absolute path.
-    error_report_path: /full/path/to/error/reports
+    error_report_path: /data-act/backend/tmp
 
     # The paths to the sample D1 and D2 files for local development
-    d1_file_path: /full/path/to/d1/file/sample/d1_sample.csv
-    d2_file_path: /full/path/to/d2/file/sample/d2_sample.csv
-
-    # We import award/subaward records from FSRS
-    fsrs:
-        procurement_service:
-            wsdl: ''  # e.g. https://example.com/?wsdl'
-            username: ''
-            password: ''
-        grant_service:
-            wsdl: ''  # e.g. https://example.com/?wsdl'
-            username: ''
-            password: ''
```

Comment:
Where did the fsrs config go? We definitely use it in scripts, why is it gone?

Reply:
I will add this back in